articles/what-is-ransomware/_index.md

```diff
@@ -0,0 +1,55 @@
+- Once the attacker receives their payment and the asymmetric ciphertext, they decrypt the data using their private key, thus successfully completing the attack.
+
+Ransomware is the type of malware that conceals its true intent––by masking itself as a “warning from the authorities.” In computing, such type of malware is called Trojan, since it misleads users by appearing unsuspicious. 
+
+### **What should you do if infected?**
+
+Here are some things you should do if you believe your device is infected:
+- First and foremost, do NOT pay the ransom.
+- Disconnect from all networks––unplug any ethernet cables, disconnect from WiFi, turn off your bluetooth, go into airplane mode.
+- Disconnect all external devices such as USB/memory sticks, phones, cameras, or external hard drives.
+- Report the incident––depending on where it occurred/on which device. If on a personal device, try the local authorities or even the FBI Internet Crime Complaint Center (IC3). If on campus, using a campus device, report it to campus security or any other authority figure.
+
+### **How can you protect yourself?**
+
+Instead of having to deal with the messy aftermath of a ransomware attack, you can take preventative measures to make sure you and your devices remain protected from such threats in the first place. Here are some things you can do:
+- If met with suspicious or unsolicited emails, do not open links or attachments within them, and do not approve or initiate downloads. 
+- This next step is important, no matter how tedious it may sound. Make a habit of regularly backing up your crucial data and information onto either an external hard drive or a USB stick. Also remember to securely disconnect said device once the backup is complete. Cloud storage is also an option––but make sure to use a server with high level encryption and multiple-factor authentication. 
+- Keep your system updated to the latest version, older versions of software may be more vulnerable to cyber attacks. 
+- Consider investing in cybersecurity software or anti-exploit technology, which can help catch and stop a threat in its tracks. 
+- Overall, just be aware of any suspicious activity in your email accounts, and on your electronic devices in general. 
+
+In the age of the internet, keeping yourself safe from potential threats to your privacy is extremely important. Regardless of how many measures you take, first staying vigilant and informed is crucial. Check out [this](https://www.section.io/engineering-education/what-is-anti-virus-software/) article on anti-malware for more information. 
+
+
```

Comment:
## Author Bio

Maitreyi is a student at the University of Colorado, pursuing a degree in Aerospace Engineering. She likes to spend her time reading, cooking, listening to music, and working on small self-started coding projects.